### ISSUE TITLE
Focus shift skips content upon editing of "Upload Files" section

### ISSUE THREAD

**Issue by zslocum1** [department-of-veterans-affairs/va.gov-team#138668]
## Background/Context

When the user starts to edit the "Upload files" section, focus is shifted to the Close button on the alert at the top of the section. However, if the user only navigates forward, the user won't hear the alert at all.

## Severity

**Approximate Section 508 Severity**:
N/A

**VA Severity**:
a11y-defect-1

**Rationale**:
Focus should be shifted to the new content (i.e. the alert), not the first interactive element here. This is also an issue on 686c and all device/browser/screen reader combinations.

## Examples

https://github.com/user-attachments/assets/ea7d1298-7a43-450f-9940-eb28e7ee7791
